sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -703,7 +703,7 @@ export type MetricChangePointFeedback = {
   /**
    * the start timestamp of feedback timerange
    */
-  startTime: Date;
+  startTime?: Date;
```

Comment:
similarly, should this be required?

Reply:
Yes they did it before but fxed it later. Thanks for pointing it, i removed the optional